src/main/java/org/gridsuite/loadflow/server/service/LoadFlowRunContext.java

```diff
@@ -21,26 +22,11 @@
  * @author Anis Touri <anis.touri at rte-france.com>
  */
 @Getter
-@Builder
-public class LoadFlowRunContext {
+public class LoadFlowRunContext extends AbstractComputationRunContext<LoadFlowParametersInfos> {
 
-    private final UUID networkUuid;
-
-    private final String variantId;
-
-    private final String receiver;
-
-    private final String provider;
-
-    private final LoadFlowParametersInfos parameters;
-
-    private final ReportContext reportContext;
-
-    private final String userId;
-
-    private final Float limitReduction;
-
-    public static LoadFlowParameters buildParameters(LoadFlowParametersInfos parameters, String provider) {
+    public static LoadFlowParameters buildParameters(LoadFlowRunContext context) {
+        LoadFlowParametersInfos parameters = context.getParameters();
+        String provider = context.getProvider();
```

Comment:
We call getters to escape this error ?
`Variable 'parameters' might not have been initialized`
There is no better way ? Maybe not but it's a bit weird because it does not protect from null values.

Reply:
Ah yes I didn't even notice it.

Reply:
I did it and it looks much better to me. Unless someone sees a good reason for this function to be static, I will keep it as a regular member of `LoadFlowRunContext`.